src/actions/User.js

```diff
@@ -90,3 +90,24 @@ export const getCurrentUser = (cookies) => {
       return json.payload;
     });
 };
+
+export const updateRecentStandards = (cardId) =>
+  fetch(urls.api.user.updateRecentStandards, {
+    method: "POST",
```

Comment:
I don't think it's a big deal, but it should technically be a `PATCH` request if you wanna be nitpicky.